src/listen.ts

```diff
@@ -86,7 +89,11 @@ export async function listen(
 
   // Local server
   let https: Listener["https"] = false;
-  const httpsOptions = listhenOptions.https as HTTPSOptions;
+  const httpsOptions = listhenOptions.ipc
+    ? false
+    : (listhenOptions.https as HTTPSOptions);
```

Comment:
AFAIK we can also support SSL via a UNIX socket. Any reason to disable?

Reply:
I had in mind, that it's a local domain, so no TLS is needed. But while thinking a bit more about, it makes sense to also support HTTPS, since "everyone" on the system can listen to that socket.

This is done and tested!